spec/support/pages/pages/sign_in_page.rb

```diff
@@ -0,0 +1,18 @@
+class SignInPage < SitePrism::Page
+  set_url'users/sign_in'
+  set_url_matcher %r{/users/sign_in}
+
+  # ELEMENTS
+  element :email_field,    '#user_email'
+  element :password_field, '#user_password'
```

Comment:
I'm not a big fan of such indentation. What if I add a new field `:super_duper_new_ultimate_field`? I need to indent all other fields and it's terrible. Let's use just one space after a comma.

Reply:
But I do a fan of it :) What should we do now? Ok, since you are the boss here I will change this. But my variant seems more readable to me. But its just a matter of taste I assume